src/App.tsx

```diff
@@ -15,48 +17,128 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
+  noSort = '',
+  byLength = 'length',
+  byName = 'alphabetically',
+}
+
+const sortGoods = (
+  goods: string[],
+  sortType: SortType,
+  isReverseGoods: boolean,
+) => {
+  if (sortType === SortType.noSort && !isReverseGoods) {
+    return goods;
+  }
+
+  const preparedGoods = [...goods];
+
+  if (sortType === SortType.byLength || sortType === SortType.byName) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortType) {
+        case SortType.byLength:
+          return good1.length - good2.length;
+        case SortType.byName:
+          return good1.localeCompare(good2);
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (isReverseGoods) {
+    preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+};
+
+export const App = () => {
```

Comment:
in my eyes, in project we have to use `export const App: React.FC`, and import React too

Reply:
@KatOlista in the latest versions of React, this is optional